package.json

```diff
@@ -1,5 +1,5 @@
 {
   "name": "@chainlink/solhint-plugin-chainlink-solidity",
-  "version": "1.0.1",
+  "version": "1.1.1",
```

Comment:
I'm not sure how this version gets bumped, the changesetbot says there's changesets but I don't see any set up for this repo

Reply:
Ah I guess we're pulling straight off master anyway. We should address it at some point in the future